web/share/js/kvm/mouse.js

```diff
@@ -70,6 +70,7 @@ export function Mouse(__getGeometry, __recordWsEvent) {
 		tools.storage.bindSimpleSwitch($("hid-mouse-reverse-scrolling-switch"), "hid.mouse.reverse_scrolling", false);
 		tools.slider.setParams($("hid-mouse-sens-slider"), 0.1, 1.9, 0.1, tools.storage.get("hid.mouse.sens", 1.0), __updateRelativeSens);
 		tools.slider.setParams($("hid-mouse-rate-slider"), 10, 100, 10, tools.storage.get("hid.mouse.rate", 100), __updateRate); // set __timer
+		tools.slider.setParams($("hid-mouse-scroll_rate-slider"), 1, 100, 1, tools.storage.get("hid.mouse.scroll_rate", 5), __updateScrollRate); 
```

Comment:
`hid-mouse-scroll_rate-slider` -> `hid-mouse-scroll-slider`

Reply:
Fixed, and fixed the scroll_rate-value as well.